src/js/site/main.js

```diff
@@ -287,6 +288,9 @@ $(document).ready(function () {
 // Currently this function is only used for the insights table
 $(highlightURLHash);
 
+// This function is used to be able to expand images when you click them
+$(expandImageOnClick);
```

Comment:
What does wrapping a function in `$(<my func>)` do? 

I'm guessing it replaces the need to use jquery's document.onready call. If this is the case, can we add this function call into one of the existing onready's, perhaps [this one](DD-319-click-to-enlarge-image)?

Reply:
I have added it here
```
// Used to call functions on document on ready
$(function() {
  // Currently this function is only used for the insights table
  highlightURLHash();
  // This function is used to be able to expand images when you click them
  expandImageOnClick();
});
```